{
  "comments": [
    {
      "key": {
        "uuid": "a50f3ed1_cb072356",
        "filename": "plugins/collect-logs/tasks/collect_host_logs.yml",
        "patchSetId": 1
      },
      "lineNbr": 111,
      "author": {
        "id": 1003043
      },
      "writtenOn": "2019-02-07T15:37:17Z",
      "side": 1,
      "message": "You are running docker Ps twice even in the good case.",
      "revId": "b4a903464701a35cdb07a52799851c75c0164cbd",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    }
  ]
}